source/state_mod.F90

```diff
@@ -231,7 +231,7 @@ module state_mod
       T_leos_ref = 19.0_r8,       &! reference T for linear eos (deg C)
       S_leos_ref = 0.035_r8,      &! reference S for linear eos (msu)
       rho_leos_ref = 1.025022_r8, &! ref dens (g/cm3) at ref T,S and 0 bar
-      alf = 2.55e-4_r8,           &! expansion coeff -(drho/dT) (gr/cm^3/K)
+      alf = 1e-4_r8,           &! 2.55 e-4r8 expansion coeff -(drho/dT) (gr/cm^3/K)
```

Comment:
I guess this matches iemic value now?

Reply:
maybe its best to introduce a new EOS type with 2nd linear EOS matching the iemic values...

Reply:
Introducing new eos type just for one coefficient is a bit strange and also that is only for benchmarks, I don't think that for global runs it will be possible to use this eos type

Reply:
ok, yea maybe you are right...in any case: why couldn't this EOS be used for global runs? (isn't IEMIC doing that?)

Reply:
Its from description of eos in pop(that it is only for special runs) and I tried linear for global run-it crash immediately

Reply:
hmm, I can't think of a physical reason why it shouldn't work...

Reply:
btw, should we revert this back to the default value or do you prefer to have the 1.e-4?

Reply:
Better to revert the value. Sure in principle that should work, but this probably force problem with kinetic energy to appear immediately and it is too simple for global runs, I don't think that we have to use it.

Reply:
ok